### PR TITLE
Basis polish and cleanup

### PR DESCRIPTION
**Pull request by wesruv**
### Responsive menu styles for basis and responsive menu style cleanup
- Added back Drop to footer
- Updated header menu styles to be secondary looking
- Added utility classes for font styling, can be really useful on blocks using 'dynamic'
- Mobile pager is now more compact for mobile and tablet
- Prefixed some menu-dropdown styles with js so they don't mess up the display of a non-JS user
- Made sure some base font sizes align with the 1/16th scale
- Hero block only uses texture if there isn't an image
- Addressed small cascade issue by moving small-text-components.css higher in cascade
- Addressed small right to left language issues
### Polishing Hero styles
- Added feature detection to basis for 'background-blend-mode' for the hero
- Added fallback if it's not supported
- Cleaned up skin.css hero styles
### Added screenshot, moved some styles out of skin.css
- Styles moved out of skin were for consistency, want to keep complex styles out of that file.

### CODE REVIEW

**Comment by wesruv** [comment]
I'll leave this alone now... promise.

**Comment by wesruv** [comment]
...ok now I'll stop touching this PR :stuck_out_tongue:

File: core/modules/system/css/menu-dropdown.theme.css [hunk 1]
```diff
@@ -48,14 +48,14 @@
   content: '-';
 }
 
-@media (min-width: 768px) {
+@media (min-width: 48em) {
```
Comment (quicksketch) [inline core/modules/system/css/menu-dropdown.theme.css:51]: Thank you for changing this, I thought that might bug you. This CSS came from the upstream library.

**Comment by quicksketch** [comment]
Test failure is a random one. I'll look into it.

This looks great! Awesome job @wesruv! Don't worry about the constant modification, that's what the feature branch is for. Merged for further review.

File: core/modules/system/css/menu-dropdown.theme.css (hunk 1 above)
Comment (wesruv) [reply core/modules/system/css/menu-dropdown.theme.css:51]: Yeup, just wanted to make sure it 'broke' at the same point that our layouts do.

I have everything on either:
25em, 34em, 48em, and 62em

25em doesn't actually show up in our layouts CSS, but I needed a smaller one, so there it is :stuck_out_tongue:

**Pull request merged**